CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ the `Marshaler<Writer>` transform overload that were _mistakenly_ passing
 `Writer` explicitly (e.g. `bond::Apply<Writer>(marshaler, value)`) will now
 get a compiler error. To fix, remove the `<Writer>` part:
 `bond::Apply(marshaler, value)`.
+* C++ codegen now generates [extern templates](http://en.cppreference.com/w/cpp/language/function_template) of `bond::Apply` instead of overloads.
```

Comment:
Even though it is unlikely to impact folks, we should mark it as breaking. Since we have other breaking changes pending, this won't be a blocker to getting this particular one released.